Gemfile

```diff
@@ -42,3 +43,5 @@ group :test do
   # Manipulate Time.now in specs
   gem 'timecop'
 end
+
+gem 'ruby_smb', git: 'https://github.com/dmohanty-r7/ruby_smb', branch: 'add_client_methods'
```

Comment:
Should we get this merged and a version merged and released in the parent repo before we merge so everything can come from rubygems?